index.html

```diff
@@ -0,0 +1,217 @@
+</head>
+<body>
+  <div class="wrapper">
+    <header class="header">
+      <div class="logo">My Logo</div>
+      <div class="hamburger-menu">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+    </header>
+    <nav class="flex-nav">
+      <div class="background-filter"></div>
+      <div class="nav-items-wrapper">
+        <ul>
+          <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
+        </ul>
+        <span class="close">+</span>
+      </div>
+    </nav>
+
+    <main class="main-content">
+      <section class="hero">
+        <h1 class="section-header">I’m Timothy Glad to see you!</h1>       
+        <p class="intro">
+          I’m a software developer! I can help you build a product, 
+          feature or website Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <span class="lets-connect">LET’S CONNECT</span>
+        <ul class="socials">
+          <li><a href=""><img src="/assets//images/vectors/twitter.svg" alt=""></a></li>
+          <li><a href=""><img src="/assets//images/vectors/Linkedin.svg" alt=""></a></li>
+          <li><a href=""><img src="/assets//images/vectors/github.svg" alt=""></a></li>
+          <li><a href=""></a><img src="/assets//images/vectors/M.svg" alt=""></li>
+          <li><a href=""><img src="/assets//images/vectors/chill.svg" alt=""></a></li>
+        </ul>
+      </section>
+      
+      <section class="portfolio" id="portfolio">
+        <div class="projects">
+          <article class="card">
+            <figure>
+              <img src="/assets/images/mobile/Snapshoot Portfolio-4.png" alt="" class="project-img">
+            </figure>
+            <h3 class="project-title">Tonic</h3>
+            <div class="explaining-spans">
+              <span>CANOPY</span>
+              <span></span>
+              <span>Back end dev</span>
+              <span></span>
+              <span>2015</span>
+            </div>
+            <p class="short-description">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <div class="badges-wrapper">
+              <span class="badge">HTML</span><span class="badge">CSS</span><span class="badge">JavaScript</span>
+            </div>
+            <button class="buttons" type="button">See Project</button>
+          </article>
+          <article class="card">
+            <figure>
+              <img src="/assets/images/mobile/Snapshoot Portfolio-1.png" alt="" class="project-img">
+            </figure>
+            <h3 class="project-title">Multi-Post Stories</h3>
+            <div class="explaining-spans">
+              <span>CANOPY</span>
+              <span></span>
+              <span>Back end dev</span>
+              <span></span>
+              <span>2015</span>
+            </div>
```

Comment:
Instead of using span here, I think it will be better to make use of `li` wrapped in the `ul` tag since we are working with lists here. 😎